rapier3d/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "dimforge_rapier3d" # Can't be named rapier3d which conflicts with the dependency.
+name = "dimforge_rapier3d"                                                   # Can't be named rapier3d which conflicts with the dependency.
```

Comment:
Big space.

Reply:
thanks! I'll try to disable that option in my extension ; for the record `"evenBetterToml.formatter.alignComments": false,`